gatsby-config.js

```diff
@@ -18,6 +18,8 @@ const googleFonts = {
     fonts: [
       'roboto\:400,700',
```

Comment:
Are we still using Robot anywhere? If not then we can remove this.

Reply:
Thanks, will remove :-)